src/components/BotMessageWithBodyInput.tsx

```diff
@@ -114,9 +116,10 @@ export default function BotMessageWithBodyInput(props: Props) {
           </Sender>
         )}
         {bodyComponent}
-        {enableEmojiFeedback && displayProfileImage && !isBotWelcomeMessage && (
-          <ReactionContainer message={message} />
-        )}
+        {enableEmojiFeedback &&
+          displayProfileImage &&
+          !isBotWelcomeMessage &&
+          !isFormMessage && <ReactionContainer message={message} />}
```

Comment:
Its minor but I'm so used to break line before operator.
https://www.flake8rules.com/rules/W504.html#:~:text=Line%20breaks%20should%20occur%20before,to%20keep%20all%20operators%20aligned.
but choose whichever that is being used in this project and keep it consistent

Reply:
so have we decided?

Reply:
Oh I missed this one. I don't mind with any and nothing is decided. I can just apply a rule for this (but separately)